root_VS2012/programs/C#/Frameworks/Infrastructure/Business/Business.csproj

```diff
@@ -89,6 +89,7 @@
     <Compile Include="Exceptions\MyBusinessApplicationExceptionMessage.cs" />
     <Compile Include="Exceptions\MyBusinessSystemExceptionMessage.cs" />
     <Compile Include="Presentation\CmnTableAdapter.cs" />
+    <Compile Include="Presentation\BaseMVController.cs" />
```

Comment:
@daisukenishino

Please include this class in **Framework.csproj**.

- BaseMVController.cs
    - You should prohibit the user from customizing.
    - You should include this class in Framework.csproj.
- MyBaseController.cs
    - You should allow the user to customize.
    - You should include this class in Business.csproj.